src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -703,4 +737,32 @@ private void commitTransaction(Connection connection) throws SQLException {
         connection.commit();
     }
 
+    private void handleSqlException(String message, SQLException ex)
```

Comment:
When you call this method, I think there is often stariway/flight ids passed to the method. I think it would be helpful when debugging if you included those id(s) in the error message. I made this comment a couple places above, but I think the same about all places this method is called.

Reply:
I changed all uses to provide stairway and flight ids if available, so we will have consistent output.